pom.xml

```diff
@@ -108,6 +110,9 @@
 		<testcontainers-version>1.19.8</testcontainers-version>
 		<hapi-structures-v24-version>2.3</hapi-structures-v24-version>
 		<jkube.generator.from>registry.access.redhat.com/ubi9/openjdk-17:1.20</jkube.generator.from>
+		<mapstruct-version>1.6.0</mapstruct-version>
```

Comment:
Can we use the mapstruct version from camel? https://github.com/jboss-fuse/camel/blob/camel-4.8.3-branch/parent/pom.xml#L332

Reply:
nope, it cannot be inherited from there.